crates/ruff/resources/test/fixtures/flake8_implicit_str_concat/ISC.py

```diff
@@ -34,3 +34,15 @@
   b"abc"
   b"def"
 )
+
+_ = """a""" """b"""
```

Comment:
Can you add a test for joining multiline strings?

Reply:
Yes, good point, done!